src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -94,15 +95,15 @@ private function configureRoutes(RoutingConfigurator $routes): void
      */
     private function getConfigDir(): string
     {
-        return $this->getProjectDir().'/config';
+        return $this->getProjectDir().'/{config}';
```

Comment:
This looks like a BC break to me if someone calls the method in their custom code.

Reply:
Ok, let's go safer with a natural is_dir check

Reply:
May I ask what the difference between `config` and `{config}` is?

Reply:
@jdreesen it makes the dir optional during glob importing